diff --git a/llvm-mca-parametric/llvm/include/llvm/Demangle/MicrosoftDemangleNodes.h b/llvm-mca-parametric/llvm/include/llvm/Demangle/MicrosoftDemangleNodes.h
index 9e3478e9..ec049cfe 100644
--- a/llvm-mca-parametric/llvm/include/llvm/Demangle/MicrosoftDemangleNodes.h
+++ b/llvm-mca-parametric/llvm/include/llvm/Demangle/MicrosoftDemangleNodes.h
@@ -5,6 +5,9 @@
 #include "llvm/Demangle/StringView.h"
 #include <array>
 
+#include <cstdint>
+#include <string>
+
 class OutputStream;
 
 namespace llvm {
@@ -602,4 +605,4 @@ struct FunctionSymbolNode : public SymbolNode {
 } // namespace ms_demangle
 } // namespace llvm
 
-#endif
\ No newline at end of file
+#endif
